packages/@aws-cdk/aws-cloudfront/lib/distribution.ts

```diff
@@ -221,6 +222,14 @@ export class Distribution extends Resource implements IDistribution {
     });
   }
 
+  private addViewerCertificate(certificate: acm.ICertificate): CfnDistribution.ViewerCertificateProperty {
+    return {
+      acmCertificateArn: certificate.certificateArn,
+      sslSupportMethod: SSLMethod.SNI,
+      minimumProtocolVersion: SecurityPolicyProtocol.TLS_V1,
```

Comment:
(Nit) - I'd prefer we opt for a higher standard by default, per the [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-distribution-viewercertificate.html#cfn-cloudfront-distribution-viewercertificate-minimumprotocolversion):

> We recommend that you specify TLSv1.2_2018 unless your viewers are using browsers or devices that don’t support TLSv1.2.

Given TLSv1.2 is supported now by all major browsers, I think it's a sane default here.